packages/amplify-provider-awscloudformation/src/delete-env.ts

```diff
@@ -16,8 +20,7 @@ async function run(context, envName, deleteS3) {
   let storageCategoryBucketName;
 
   if (deleteS3) {
-    const projectDetails = context.amplify.getProjectDetails();
-    deploymentBucketName = projectDetails.teamProviderInfo[envName][ProviderName].DeploymentBucketName;
+    deploymentBucketName = stateManager.getTeamProviderInfo()?.[envName]?.[ProviderName]?.DeploymentBucketName;
```

Comment:
Does this need to be updated?

Reply:
not yet. This is reading from the provider block